src/controllers/player.coffee

```diff
@@ -17,14 +17,25 @@ angular.module 'matching'
 	$scope.totalItems = 0
 	$scope.setCreated = false
 	$scope.helpVisible = false
+	$scope.completePerPage = []
 
 	$scope.unfinishedPagesBefore = false
 	$scope.unfinishedPagesAfter = false
+	helpModal = document.getElementById("instructions");
```

Comment:
It took a while to figure out, but the test suite is failing because of these references to DOM objects. Jest is running the player code in node, where the DOM isn't actually present - so any attempts to reference those DOM elements will fail (such as applying or removing the `inert` attribute.

There are two potential solutions to this that I can think of:
a) Any lines that are applying or removing the `inert` attribute should conditionally check for the existence of the DOM element first
b) We use another package like `jsdom` to simulate the DOM elements when running tests. Copilot can help you set that up.